include/tune_quda.h

```diff
@@ -602,6 +602,21 @@ namespace quda {
    */
   void setPolicyTuning(bool);
 
+  /**
+   * @brief Query whether we are currently tuning a policy
+   */
+  bool policyTuning();
+
+  /**
+   * @brief Enable / disable whether we are tuning an uber kernel
+   */
+  void setUberTuning(bool);
```

Comment:
Why do we need a separate `uberTuning`?  What function does this serve that `policyTuning` doesn't?

Reply:
The uber kernel is not a policy but needs to run on all GPUs so that flags are set correctly. I added that to avoid an infinite spin at some point but good to reconsider whether we can get away without it.

Reply:
Yeh, I realize that this isn't a policy and is a separate use case.  This may be useful in the future for other uber kernels that aren't part of a policy at all, e.g., if we were to implement the gauge force as an uber kernel for example, or more conservatively, implementing the gauge field extension using NVSHMEM.  So this is good to keep.

Reply:
I think we should include @mathiaswagner 's reply here for clarity. An inline comment that explicitly says "An uber kernel is not a policy but it needs to run on all GPUs so that flags are set correctly." would be a useful clarifier here and elsewhere.